src/tests/fixtures/form.html

```diff
@@ -25,6 +25,18 @@ <h1>Form</h1>
         <input type="hidden" name="greeting" value="Hello from a redirect">
         <input id="standard-get-form-submit" type="submit" value="form[method=get]">
       </form>
+      <hr>
+      <form>
+        <button id="form-action-none-q-a" name="q" value="a">Submit ?q=a to form:not([action])</button>
+      </form>
+      <form action="/src/tests/fixtures/form.html?sort=asc">
+        <button id="form-action-self-submit">Submit form[action]</button>
```

Comment:
I've just discovered that this is still slightly incorrect behavior.

I tested with an application without Turbo or any other JavaScript, and submitting a form with an `[action]` value that contains query parameters requests that `[action]` path without any key-value pairs (that is, with `?` as its the value of `URL.search`):

```html
<form action="/src/tests/fixtures/form.html?sort=asc">
  <!-- clicking the button navigates to `/src/tests/fixtures/form?` -->
  <button>Submit</button>
</form>
```

I'm in the process of adding test coverage and opening a separate PR.